flow/size.yaml

```diff
@@ -62,3 +67,70 @@ components:
           type: integer
           default: 1518
           x-field-uid: 2
+    Flow.SizeWeightPairs:
+      type: object
+      description: |-
+        Frame size distribution, defined as <size, weight> pairs (including IMIX distribution).
+        Frames are randomly generated such that the proportion of each frame size out of the total number of frames 
+        are matching with the weight value of the <size, weight> pair. However, as with any other probability 
+        distribution, the sample distribution is close to theoretical value only if the size of the sample is reasonably large. 
+        When the number of frames is very low the transmitted frames may not come close to the ratio described in the weight.
+      properties:
+        choice:
+          type: string
+          default: predefined
+          x-field-uid: 1
+          x-enum:
+            predefined:
+              x-field-uid: 1
+            custom:
+              x-field-uid: 2
+        predefined:
+          type: string
+          default: imix
+          description: |-
+            Specify predefined frame size distribution <size, weight> pairs (including IMIX distribution).  
+            The available predefined distribution pairs are:
+            - IMIX (64:7, 570:4, and 1518:1)  
+            - IPSec IMIX (90:58.67, 92:2, 594:23.66 and 1418:15.67)  
+            - IPv6 IMIX (60:58.67, 496:2, 594:23.66 and 1518:15.67)  
+            - Standard IMIX (58:58.67, 62:2, 594:23.66 and 1518:15.67)  
+            - TCP IMIX (90:58.67, 92:2, 594:23.66 and 1518:15.67)  
+          x-field-uid: 2
+          x-enum:
+            imix:
+              x-field-uid: 1
+            ipsec_imix:
```

Comment:
is this supported by some implementations?

Reply:
In IxNetwork, supported in IXHW and IxVM. In OTG, it will be supported in IXHW and Ixia-C SW ports. UHD support is under discussion.